NAMESPACE

```diff
@@ -1,5 +1,11 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(epidist,epidist_ltcad)
```

Comment:
Issue to finalise name?

Reply:
Yes agree. Will make.

Reply:
https://github.com/epinowcast/epidist/issues/72